Formula/jpeg-xl.rb

```diff
@@ -0,0 +1,34 @@
+class JpegXl < Formula
+  desc "New file format for still image compression"
+  homepage "https://jpeg.org/jpegxl/index.html"
+  url "https://gitlab.com/wg1/jpeg-xl.git",
+    tag:      "v0.2",
+    revision: "31c71b0f61123a40789b0b8f54feb70e5995420e"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "gcc"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "openexr"
+
+  fails_with :clang
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", "-DBUILD_TESTING=OFF", *std_cmake_args
+      system "cmake", "--build", "."
+    end
+    bin.install "build/tools/benchmark_xl" => "benchmark_xl"
+    bin.install "build/tools/cjxl" => "cjxl"
+    bin.install "build/tools/djxl" => "djxl"
```

Comment:
```suggestion
      system "cmake", "--build", "."
      system "cmake", "--build". ".", "--target", "install"
    end
```
Try this to see if it works.